fbchat/graphql.py

```diff
@@ -109,6 +109,12 @@ def graphql_to_attachment(a):
             large_image=a.get('large_image'),
             uid=a.get('legacy_attachment_id')
         )
+    elif _type == 'MessageAudio':
+        return AudioAttachment(
```

Comment:
The `a` dict contains an `audio_type`, maybe we should include this in the `AudioAttachment` too?